buildscripts/config/config_orion.sh

```diff
@@ -31,8 +33,10 @@ export   MAKE_CLEAN=N
 export DOWNLOAD_ONLY=F
 export STACK_EXIT_ON_FAIL=T
 export WGET="wget -nv"
+
 #Global compiler flags
 export FFLAGS=""
 export CFLAGS=""
-export CXXFLAGS="-std=c++14"
-export LDFLAGS="-std=c++14"
+# These don't work for some of the packages (when the OS GNU compiler is used)
```

Comment:
There are some packages that don't use icc but gcc. For orion, that defaults to gcc 4.8.5 ...